src/localsocketcontroller.h

```diff
@@ -33,6 +37,9 @@ class LocalSocketController final : public ControllerImpl {
 
   void cleanupBackendLogs() override;
 
+ public slots:
```

Comment:
remove `public slots`. Passing a function pointer doesn't require the `slot` keyword.